buddyboss/buddyboss-platform
# mentions in groups and forums to only members of that group

### All Submissions:






### Changes proposed in this Pull Request:

Mention only group members in group and forums.
https://files.slack.com/files-pri/T0VNCRXSL-F011LFTPW3G/image_from_ios.jpg

### How to test the changes in this Pull Request:

1. Go to group > feed
2. try to mention in activity feed
3. It should only allow members of that group.

### Other information:




### Changelog entry

> Mentions - groups and forums mentions results restricted to members of group.
